lib/jekyll/commands/new.rb

```diff
@@ -21,6 +22,12 @@ def init_with_program(prog)
         def process(args, options = {})
           raise ArgumentError, "You must specify a path." if args.empty?
 
+          if args.size == 1
+            new_blog_title = args[0]
```

Comment:
The kind of madness where a `new_blog_title` of "blog" should be converted to "Blog". You could get into all sorts of inane arguments about what is an acceptable directory name (because Unicode), however, I am of the opinion that all representations of that name should be uppercase by default. If people want to change it, fine, there's nothing stopping you.